lib/smart_answer_flows/marriage-abroad/outcomes/_synonyms_of_cp.govspeak.erb

```diff
@@ -114,7 +114,7 @@
   - ’domestic partnership’ (in Nova Scotia)
   - ’union civile’, or ’civil union’ (in Quebec)
 <% when 'new-zealand' %>
-  You may be able to get married or enter into a ‘civil union’ at the British embassy or consulate in New Zealand. Civil unions are recognised as civil partnerships under British law.
+  Civil unions are recognised as civil partnerships under British law.
```

Comment:
@sistephens: That'd be brilliant. In this case the change is so small, it's not a big deal. However, in general explaining the *reason* behind a change the *most* valuable thing to put in a commit note. It's usually possible to work out *what* has changed by looking at the changes, but it's impossible to work out *why*. If the information exists elsewhere, it's probably sufficient to link to them, as long as you're confident they're not going to disappear.